modules/Authentication.php

```diff
@@ -94,14 +116,21 @@ public static function createSession($db, $user_id)
         return false;
     }
 
-    // Generate random session token
+    /**
+     * @method : generateSessionToken
+     * @description : Generate random session token
```

Comment:
The description could be like this: "Generates random session token to be used for <purpose>"